editor/src/components/editor/canvas-toolbar.tsx

```diff
@@ -168,22 +168,16 @@ export const CanvasToolbar = React.memo(() => {
       onClick={stopPropagation}
     >
       <FlexColumn style={{ padding: 4 }}>
-        <header style={{ paddingLeft: 4, fontSize: 10, fontWeight: 500 }}>Tools</header>
+        <header style={{ paddingLeft: 4, fontSize: 10, fontWeight: 500 }}>Scale</header>
         <FlexRow style={{ flexWrap: 'wrap', gap: 4, padding: 4 }}>
-          <Tooltip title='Select' placement='bottom'>
-            <InsertModeButton
-              iconType='pointer'
-              iconCategory='tools'
-              onClick={clickSelectModeButton}
-            />
-          </Tooltip>
-          <Tooltip title='Insert text' placement='bottom'>
-            <InsertModeButton
-              iconType='pure-text'
-              primary={spanInsertion}
-              onClick={insertSpanCallback}
-            />
+          <Tooltip title='Zoom in' placement='bottom'>
```

Comment:
The tooltip here should say something different, like "reset zoom" or something like that